src/Sylius/Component/Core/spec/Updater/UnpaidOrdersStateUpdaterSpec.php

```diff
@@ -24,9 +25,9 @@
 
 final class UnpaidOrdersStateUpdaterSpec extends ObjectBehavior
 {
-    function let(OrderRepositoryInterface $orderRepository, Factory $stateMachineFactory): void
+    function let(OrderRepositoryInterface $orderRepository, Factory $stateMachineFactory, LoggerInterface $logger): void
     {
-        $this->beConstructedWith($orderRepository, $stateMachineFactory, '10 months');
+        $this->beConstructedWith($orderRepository, $stateMachineFactory, '10 months', $logger);
```

Comment:
Can we make a unit test for behavior without logger?

Reply:
Hey @lchrusciel,

I added the `it_wont_stop_cancelling_unpaid_orders_on_exception_for_a_single_order_and_skips_logging_if_logger_is_not_set` (I am not sure if the name is meaningful enough or too long) test. I hope that's okay. Cheers

Reply:
Hey @twojtylak, it is understandable enough ;)